api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -42,10 +42,11 @@
      "name": "stdout",
```

Comment:
<p>This output looks a little more bizarre now. Is this expected due to multiple sources providing yield values?</p><p> </p><p>Same for the first visualized output.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Yes, it's sources reporting very different values. I filtered out PS&amp;D, and that largely cleans it up since they're on a different calendar from the other three sources. I also put a comment in the code explaining that being excluded.</p>